PowerQueryM/README.md

```diff
@@ -0,0 +1,16 @@
+This folder has three sample Power Query M queries that can be used to load MCast™ data into Excel or PowerBI.
+
+The first sample, ObservedLoad.pq, uses the MCast™ API to populate a table with the last 60 days of consumption data. The second sample, ForecastedLoad.pq, uses the MCast™ API to populate a table with all of the pinned one-day-ahead MCast™ forecast values from the last 60 days. The third sample, APE.pq, merges the results of the previous two queries and adds columns for the day of week and the absolute percent error.
+
+To run these samples, open Excel and in the "Data" tab, select "Get Data" -> "From Other Sources" -> "Blank Query" to create a new query. 
+
+![BlankQuery]
```

Comment:
You can click the preview button on this PR (or even outside the PR, just select this branch and then navigate to the PowerQueryM folder) to see this file as it actually renders.